submissions/fiyingabriel/aboutMe/PRD for E-Commerce Using HTML.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE HTML>
```

Comment:
Styling is not part of the criteria for this project. I think you should remove all styling added to this file.